default.nix

```diff
@@ -83,7 +83,7 @@ let commonBuildInputs = pkgs:
     pkgs.ocamlPackages.cow
     pkgs.ocamlPackages.num
     pkgs.ocamlPackages.stdint
-    pkgs.ocamlPackages.wasm
+    pkgs.ocamlPackages.wasm_1
```

Comment:
the normal `wasm` was 2.0 for quite some time, but adopting it needs changes on our side. We can do experiments, tough, removing the `_1` now.